app/controllers/api/v1/notifications_controller.rb

```diff
@@ -81,6 +81,10 @@ def exclude_types
     val
   end
 
+  def from_account
+    params[:from_account]
```

Comment:
While the internal name is `from_account_id`, perhaps it would make more sense to name the query parameter simply `account_id`, since the "from" seems to be redundant in the context of how this will be used.

Reply:
done